### ISSUE TITLE
No source code is available for type java.util.BitSet with gwt.2.8.0-beta1

### ISSUE THREAD

**Issue by RRosss** [google/guava#2547]
I this morning mvn downloaded updates for guava-gwt-20.0-SNAPSHOT.

If I compile gwt project with gwt.version 2.8.0-beta1 I get this error.

```
Compiling module app.Application
[INFO]    Tracing compile failure path for type 'com.google.common.collect.BinaryTreeTraverser'
[INFO]       [ERROR] Errors in 'jar:file:/Users/ros/.m2/repository/com/google/guava/guava-gwt/20.0-SNAPSHOT/guava-gwt-20.0-SNAPSHOT.jar!/com/google/common/collect/BinaryTreeTraverser.java'
[INFO]          [ERROR] Line 134: No source code is available for type java.util.BitSet; did you forget to inherit a required module?
[INFO]    [ERROR] Aborting compile due to errors in some input files
```

With gwt 2.8.0-rc1 my project compile but can't upgrade gwt version.

**Comment by cpovirk** [comment]
Guava 20 is going to require GWT 2.8.0 (rc1 or later). We could conceivably back out some recent changes so that it works with beta1, but it feels weird to maintain compatibility with a beta for the long term, especially when that beta is known to have problems with Guava: https://github.com/gwtproject/gwt/issues/9351#issuecomment-228155079. Can you tell us what blocks you from updgrading to rc1? Is it just a bug that has hope of being fixed before the final 2.8.0?

**Comment by RRosss** [comment]
I agree with you.
We have a problem with SDM witch fails during the first compilation and the problem seems in the generation of the source maps compiling with  GWT 2.8.0 rc1 (and rc2)(https://groups.google.com/forum/#!topic/google-web-toolkit/ATR3ZWcVrEs) 
We also hope they'll fix with final 2.8.0 but in the mean time we are stuck.
